### DISCUSSION TITLE
Taro组件生命周期问题

### DISCUSSION THREAD

**Discussion by Arbureva** [NervJS/taro#12707] [Q & A]
Taro + vue3 能不能像原生小程序开发那样 自动销毁页面 重新进入就重新加载 而不是把已经加载好的东西再拿出来 里面的变量啥的都得手动销毁吗？写个前端跟写C++一样 太难受了！！！！

**Comment by Chen-jj** [comment]
@Locter9001 页面的销毁与回收是跟小程序一样，不理解你遇到了什么情况

**Reply by Arbureva** [reply]
> @Locter9001 页面的销毁与回收是跟小程序一样，不理解你遇到了什么情况

当我将数据传递到组件 然后离开当前页面再次进入时 他的数据还在

**Reply by AdvancedCat** [reply]
> 当我将数据传递到组件 然后离开当前页面再次进入时 他的数据还在

“离开 & 再次进入”有多种情况存在的，可以进一步描述一下操作步骤；

或者，可以提起 issue ，并提供一下可复现的最小DEMO，可以帮助我们快速定位问题

**Reply by llh1187** [reply]
> > @Locter9001 页面的销毁与回收是跟小程序一样，不理解你遇到了什么情况
> 
> 当我将数据传递到组件 然后离开当前页面再次进入时 他的数据还在

你是不是说tabBar页？

**Reply by Arbureva** [reply]
> > > @Locter9001 页面的销毁与回收是跟小程序一样，不理解你遇到了什么情况
> > 
> > 
> > 当我将数据传递到组件 然后离开当前页面再次进入时 他的数据还在
> 
> 你是不是说tabBar页？

不是，之前写的时候在其它页面也出现了这样的问题，不过现在项目已经完成了，我用其他方式完成了，也没管太多，但印象很深之前确实出现了这样的问题，可能是组件状态没被销毁还是怎么滴
